numba/core/typing/templates.py

```diff
@@ -500,7 +500,24 @@ def generic(self, args, kws):
             # need to run the compiler front end up to type inference to compute
             # a signature
             from numba.core import typed_passes, compiler
-            ir = compiler.run_frontend(disp_type.dispatcher.py_func)
+            from numba.core.inline_closurecall import InlineWorker
+            fcomp = disp._compiler
+            flags = compiler.Flags()
+
+            # Updating these causes problems?!
+            #fcomp.targetdescr.options.parse_as_flags(flags,
+            #                                         fcomp.targetoptions)
+            #flags = fcomp._customize_flags(flags)
```

Comment:
Is this a problem? Do flags influence IR gen?

Reply:
only in `cfuncs` it seems

Reply:
leave it for now?